pubsub/gcloud/aio/pubsub/subscriber.py

```diff
@@ -149,8 +157,9 @@ async def maybe_ack(ack_id: str) -> None:
                     ack_ids = []
 
                 log.warning(
-                    'ack request failed, better luck next batch',
+                    'ack request failed',
```

Comment:
💟 bahahaah yaaaa ... it was a bit ... passive aggressive 🇨🇦🇨🇦🇨🇦 sorry

Reply:
Hahaha yeah, some of this was just "lets make our error messages a bit less cutesy" :P